src/plugins/vis_augmenter/public/vega/helpers.ts

```diff
@@ -0,0 +1,324 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import moment from 'moment';
+import { cloneDeep, isEmpty, get } from 'lodash';
+import {
+  OpenSearchDashboardsDatatable,
+  OpenSearchDashboardsDatatableColumn,
+} from '../../../expressions/public';
+import {
+  PointInTimeEvent,
+  PointInTimeEventsVisLayer,
+  VIS_LAYER_COLUMN_TYPE,
+  EVENT_COLOR,
+  EVENT_MARK_SIZE,
+  EVENT_MARK_SIZE_ENLARGED,
+  EVENT_MARK_SHAPE,
+  EVENT_TIMELINE_HEIGHT,
+  HOVER_PARAM,
+} from '../';
+
+export const enableEventsInConfig = (config: { kibana: {} }) => {
+  return {
+    ...config,
+    kibana: {
+      ...config.kibana,
+      showEvents: true,
+    },
+  };
+};
+
+// Get the first xaxis field as only 1 setup of X Axis will be supported and
+// there won't be support for split series and split chart
+export const getXAxisId = (
+  dimensions: any,
+  columns: OpenSearchDashboardsDatatableColumn[]
+): string => {
+  return columns.filter((column) => column.name === dimensions.x.label)[0].id;
+};
```

Comment:
I was running into cyclical dependency issues when importing functions from vis augmenter plugin and visualizations plugin, and vice versa, which is why it is re-declared here. Prefer to leave it since it is only this single minor/simple fn.